composer.json

```diff
@@ -22,7 +22,8 @@
         "queue-interop/amqp-interop": "^0.5",
         "doctrine/common": "^2.3",
         "doctrine/dbal": "^2.2",
-        "aws/aws-sdk-php": "^2.8"
+        "aws/aws-sdk-php": "^2.8",
+        "sentry/sentry": "^1.6"
```

Comment:
No, I change the constaint to "=1.5.0" (same as [official symfony bundle](https://github.com/getsentry/sentry-symfony))